app/controllers/admin/invitations_controller.rb

```diff
@@ -11,6 +11,13 @@ def new; end
     def create
       email = params.dig(:user, :email)
       name = params.dig(:user, :name)
+
+      if User.exists?(["lower(email) = ? AND registered = ?", email, true])
```

Comment:
@Zhao-Andy I can't verify this with Blazer because we excluded the email column for privacy reasons, but using `lower()` here almost certainly means the index won't be used and we end up with a seq scan.

Given that we have `before_validation :downcase_email` in `user.rb.` we're probably better of doing this:

```ruby
if  User.exists?(email: email.downcase, registered: true)
```

Reply:
Added via https://github.com/forem/forem/pull/12537